meta-client/src/main/java/org/apache/helix/metaclient/constants/MetaClientKeeperException.java

```diff
@@ -0,0 +1,39 @@
+package org.apache.helix.metaclient.constants;
+
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+public final class MetaClientKeeperException extends MetaClientException {
```

Comment:
I think we probably want more specific exception than just keeper exception. 
Helix defined  ZkException translate KeeperException to ZkException, and metaclient translates ZkException to metaclient defined exception. 
you can do 
```
   try {
     //........ 
    } catch (KeeperException e) {
      throw translateZkExceptionToMetaclientException(ZkException.create(e));
    }
```

Reply:
In this case, it wouldn't be necessary to create a separate MetaClientKeeperException then? A ZkException then throws a generic MetaClient exception